src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/SilentLevel.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  The MIT License
+ *
+ *  Copyright 2014 Christian Georgi
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package com.sonyericsson.hudson.plugins.gerrit.trigger.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Values for Silent Mode.
+ */
+public enum SilentLevel {
+
+    /**
+     * Don't disclose anything.
+     */
+    ALL,
+
+    /**
+     * Keep comments to yourself.
+     */
+    COMMENTS,
+
+    /**
+     * Keep votes to yourself.
+     */
+    VOTES;
+
+
+    /**
+     * Method to get the list of silent levels.
+     * @return A list of silent levels.
+     */
+    public static List<String> getLevels() {
+      List<String> levels = new ArrayList();
+      levels.add(ALL.toString());
```

Comment:
I'm guessing you are using this for UI diaply, in that case it should be the I18n versions of the names.